dropwizard-lifecycle/src/main/java/io/dropwizard/lifecycle/setup/ScheduledExecutorServiceBuilder.java

```diff
@@ -1,59 +1,73 @@
 package io.dropwizard.lifecycle.setup;
 
-import com.google.common.util.concurrent.ThreadFactoryBuilder;
-import io.dropwizard.lifecycle.ExecutorServiceManager;
-import io.dropwizard.util.Duration;
-
 import java.util.concurrent.RejectedExecutionHandler;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ScheduledThreadPoolExecutor;
 import java.util.concurrent.ThreadFactory;
 import java.util.concurrent.ThreadPoolExecutor;
 
+import com.google.common.util.concurrent.ThreadFactoryBuilder;
+
+import io.dropwizard.lifecycle.ExecutorServiceManager;
+import io.dropwizard.util.Duration;
+
 public class ScheduledExecutorServiceBuilder {
+
     private final LifecycleEnvironment environment;
     private final String nameFormat;
     private int poolSize;
     private ThreadFactory threadFactory;
     private Duration shutdownTime;
     private RejectedExecutionHandler handler;
+    private Boolean removeOnCancel;
```

Comment:
We could use a boolean primitive here with default value of `false`, because an executor by default doesn't have the "Remove on cancel" policy.